ISS/rforms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms            
+from django.contrib.auth.models import User   
+from django.contrib.auth.forms import UserCreationForm      
+
+class MyRegistrationForm(UserCreationForm):
```

Comment:
Nit pick, but something like PosterRegistrationForm or just RegistrationForm (if that doesn't clash with something else) reads a little more cleanly. Remember a rename requires a change in the view (user_reg.py)